modules/administration/pages/virtual-hosts.adoc

```diff
@@ -111,4 +111,10 @@ If you do not refresh the data from a new Virtual Host Manager, host data will n
 ====
 +
 
-. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems] .
+. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems].
+
+[NOTE]
+====
+Depending on your environment and settings, you might receive an ``invalid certificate`` error when you connect to the server.
+If this occurs, check that your SSL certificate is current, run the [command]``update-ca-certificates`` command on the command line to re-trust your certificate, and restart the spacewalk services.
```

Comment:
Also, I tested, and it worked for me *without* restarting the spacewalk services.

Reply:
I think it's probably best to leave it in. If they do it and don't need to, then no harm done.